perfmetrics/scripts/continuous_test/ml_tests/tf/README.md

```diff
@@ -0,0 +1,28 @@
+# Kokoro test for tf model reliability using gcsfuse
+
+This readme contains file descriptions, libraries and model used for the test.
+
+## Test description:
+
+* We have used tf2.10 framework along with tf-model-garden library (v2.10) 
```

Comment:
Create sections as mentioned over https://github.com/GoogleCloudPlatform/gcsfuse/pull/878#discussion_r1081020461. 
Also, add links to libraries and packages required.